installer/scripts/common/setupKeptn.sh

```diff
@@ -124,4 +124,7 @@ cat ../manifests/keptn/keptn-api-virtualservice.yaml | \
   sed 's~DOMAIN_PLACEHOLDER~'"$DOMAIN"'~' > ../manifests/keptn/gen/keptn-api-virtualservice.yaml
 
 kubectl apply -f ../manifests/keptn/gen/keptn-api-virtualservice.yaml
-verify_kubectl $? "Deploying keptn api virtualservice failed."
\ No newline at end of file
+verify_kubectl $? "Deploying keptn api virtualservice failed."
+
+# Inject istio into jmeter-service
+kubectl get deployment -n keptn jmeter-service -oyaml | istioctl kube-inject -f - | kubectl apply -f -
```

Comment:
If we ever use another service, e.g., "neoload", we would have to rewrite this.

I'm wondering if this should be somewhere in the installation of jmeter, not in the installation of keptn?
@johannes-b @dirkwall @michaelHampel @agrimmer any thoughts?